src/components/Nav/Menu.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import { navItems } from '../../routes';
+import { matchPath } from 'react-router';
+import { Link } from 'react-router-dom';
+import { Nav, NavList, NavItem, PageSidebar } from '@patternfly/react-core';
+import _ from 'lodash';
+
+const ExternalLink = ({ href, name }) => (
+  <NavItem isActive={false} key={href}>
+    <a className="pf-c-nav__link" href={href} target="_blank">
+      {name}
+      <span className="co-external-link" />
+    </a>
+  </NavItem>
+);
+
+type MenuProps = {
+  isNavOpen: boolean;
+  location: any;
+  jaegerUrl: string;
+  jaegerIntegration: boolean;
+};
+
+type MenuState = {
+  activeItem: string;
+};
+
+class Menu extends React.Component<MenuProps, MenuState> {
+  static contextTypes = {
+    router: () => null
+  };
+
+  constructor(props: MenuProps) {
+    super(props);
+    this.state = {
+      activeItem: 'Overview'
+    };
+  }
+
+  goTojaeger() {
+    window.open(this.props.jaegerUrl, '_blank');
+  }
+
+  renderMenuItems = () => {
```

Comment:
mmm let me check this it seems that in the last version this component changed to require the key, Thanks @mtho11

Reply:
Mmm i don't see this error in the console...only this
```bash
index.js:2178 Warning: Please use `require("history").createMemoryHistory` instead of `require("history/createMemoryHistory")`. Support for the latter will be removed in the next major release.
```
but it's not releated with this PR anyway I'll set a key.

Reply:
Can you check that there is not error now @mtho11 ?